docs/user-guide/advanced_topics/monte_carlo.md

```diff
@@ -0,0 +1,23 @@
+# Monte Carlo Simulations
+
+```eval_rst
+.. toctree::
+    :maxdepth: 1
+
+
+```
```

Comment:
`eval_rst` no longer works with the new markdown parser. `{eval-rst}` is the keyword for the directive.

```suggestion
```{eval-rst}
.. toctree::
    :maxdepth: 1
```
```